instrumentation/log4j/log4j-appender-2.16/javaagent/build.gradle.kts

```diff
@@ -0,0 +1,45 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+muzzle {
+  pass {
+    group.set("org.apache.logging.log4j")
+    module.set("log4j-core")
+    versions.set("[2.11.0,)")
+    assertInverse.set(true)
+  }
+}
+
+dependencies {
+  library("org.apache.logging.log4j:log4j-core:2.16.0")
+
+  compileOnly(project(":instrumentation-api-appender"))
+
+  implementation(project(":instrumentation:log4j:log4j-appender-2.16:library"))
+
+  testImplementation("org.awaitility:awaitility")
+
+  testImplementation("com.lmax:disruptor:3.4.2")
```

Comment:
Why this is needed?

Reply:
I added a comment to explain, it's needed for the async logger test